src/utils/text.ts

```diff
@@ -7,3 +7,15 @@ export function truncateText(text: string, length: number) {
 export function copyToClipboard(text: string) {
   navigator.clipboard.writeText(text)
 }
+
+export function isTextContainsOnlyEmoji(text: string) {
+  const emojiRegex = /\p{Extended_Pictographic}/gu
+  const isOnlyEmoji = text.replace(emojiRegex, '').trim().length === 0
+  return isOnlyEmoji
+}
+
+export function getEmojiAmount(text: string) {
+  const emojiRegex = /\p{Extended_Pictographic}/gu
```

Comment:
extract regex in one constant

Reply:
Done in [30a5ae5](https://github.com/dappforce/grillchat/pull/39/commits/30a5ae593232530e6be665aadd7d5ab3a6e9846d)